includes/footer.php

```diff
@@ -1,22 +1,141 @@
+                        <li data-target="#carouselSponsor" data-slide-to="11"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="12"></li>
+                    </ol>
+
+                    <div class="carousel-inner" role="listbox">
+                         <div class="item active">
+                            <img src="/img/sponsors/ehs.jpg" alt="Eastlake High School Logo" title="Eastlake High School"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/google.jpg" alt="Google Logo" title="Google"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/lwsd.jpg" alt="Lake Washington School Foundation Logo" title="Lake Washington School Foundation"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/boeing.jpg" alt="Boeing Logo" title="Boeing"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/chase.jpg" alt="Chase Logo" title="Chase"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/FIRSTWa.jpg" alt="FIRST Washington Logo" title="FIRST Washington"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/godaddy.jpg" alt="GoDaddy Logo" title="GoDaddy"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/honeywell.jpg" alt="Honeywell Logo" title="Honeywell"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/kenworth.jpg" alt="Kenworth Logo" title="Kenworth"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/microsoft.jpg" alt="Microsoft Logo" title="Microsoft"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/ospi.jpg" alt="Office of the Superintendant of Public Instruction Logo" title="Office of the Superintendent of Public Instruction"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/rocketdyne.jpg" alt="Aerojet Rocketdyne Logo" title="Aerojet Rocketdyne"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/sae.jpg" alt="SAE Washington Logo" title="SAE Washington"/>
+                        </div>
                     </div>
-                    <hr>
+
+                    <a href="#carouselSponsor" role="button" id="playpause" onClick="$('#carouselSponsor').carousel('pause');">
```

Comment:
Pause button currently only pauses, and never resumes. Need to make a JS function to keep track of its state (see`index.php`).